tools/n64tool.c

```diff
@@ -39,33 +40,41 @@
 /* Easier to write from here */
 static int title[TITLE_SIZE];
 static int wrote_title = 0;
-static unsigned char zero[1024] = {0};
+static unsigned char zero[1024];

```

Comment:
Actually, this is guaranteed by C standard. If you initialize less elements, the rest of the array is guaranteed to be zero initialized. It's not a latent bug, it's the way C works.

Reply:
True. Testing with:
```
void sink(void *);

int test()
{
    int arr[16] = {0};
    sink(arr);
}
```
will output
```
test:
.LFB0:
	.cfi_startproc
	pushq	%rbp
	.cfi_def_cfa_offset 16
	.cfi_offset 6, -16
	movq	%rsp, %rbp
	.cfi_def_cfa_register 6
	subq	$64, %rsp
	movq	$0, -64(%rbp)
	movq	$0, -56(%rbp)
	movq	$0, -48(%rbp)
	movq	$0, -40(%rbp)
	movq	$0, -32(%rbp)
	movq	$0, -24(%rbp)
	movq	$0, -16(%rbp)
	movq	$0, -8(%rbp)
	leaq	-64(%rbp), %rax
	movq	%rax, %rdi
	call	sink@PLT
	nop
	leave
```
Therefore, the array was indeed initialized with 0.